Standards/CCSMB-12.md

```diff
@@ -0,0 +1,90 @@
+# *CCSMB 12:* 3D JSON Array Format
+
+*Author: Dimaguy*
+
+*Version: 1.0.0*
+
+*Last revised: 2024-04-02*
+
+The words MUST, MUST NOT, SHOULD, SHOULD NOT, and MAY are defined in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+This RFC introduces the 3D JSON Array Format, used for printing 3D Blocks in ComputerCraft using the mod [sc-peripherals](https://github.com/SwitchCraftCC/sc-peripherals)
+
+## Quick information
+
+| Information |                           |
+| ----------- | ------------------------- |
+| Version     | 1.0.0                     |
+| Type        | Model Array file format   |
+| MIME        | `model/3dja`              |
+| Extensions  | `.3dja`                   |
+
+## Technical details
+
+The file MUST be composed of a single JSON object ([RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259), "The JavaScript Object Notation (JSON) Data Interchange Format") with the following ruleset:
```

Comment:
Consider moving the citation to [a Markdown footnote](https://tiiny.host/blog/markdown-footnotes/).

Reply:
Moved